src/GoogleAdWordsEventForwarder.js

```diff
@@ -567,7 +565,7 @@
                                 consentMappings,
                                 consentPayloadDefaults,
                             );
-                        consentPayloadAsString = stringifyPayload(defaultConsentPayload);
+                        consentPayloadAsString = JSON.stringify(defaultConsentPayload);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify shows 

```
Properties are visited using the same algorithm as [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys), which has a well-defined order and is stable across implementations. For example, JSON.stringify on the same object will always produce the same string, and JSON.parse(JSON.stringify(obj)) would produce an object with the same key ordering as the original (assuming the object is completely JSON-serializable).
```

Reply:
This is a great callout, but in our case, we're doing a very simple comparison and the keys are pre-defined. Also, as mentioned, if we happen to send a duplicate consent update, it will likely be ignored.